### DISCUSSION TITLE
Does skywalking support zabbix agent2

### DISCUSSION THREAD

**Discussion by shilin66** [apache/skywalking#10867] [Q&A]
Zabbix has two kinds of agent ： zabbix agent and **zabbix agent2**

When i use zabbix agent , it's no problem.

But when i use **zabbix agent2**, it had some problem

this is zabbix agent2 log
![image](https://github.com/apache/skywalking/assets/39396378/4ca43fb6-1fd1-493f-8882-14426f1007e4)

this is skywalking log
![image](https://github.com/apache/skywalking/assets/39396378/bfeccb20-cddf-4262-9267-52ff3dc6178f)

this is skywalking config
![image](https://github.com/apache/skywalking/assets/39396378/68d17201-173d-49b2-8b5f-4fa32584e2ff)

Thanks for help!

**Comment by wu-sheng** [comment]
I don't think we have the new agent support.
